pyproject.toml

```diff
@@ -19,6 +19,7 @@ aioresponses = "^0.7.6"
 tiktoken = "^0.6.0"
 openai = "^0.27.8"
 pytest-mock = "^3.14.0"
+sentencepiece = "^0.2.0"
```

Comment:
Just wondering if you also need to set min version for `google-cloud-aiplatform` to 1.57.0 ?

Reply:
They are even writing about 1.60.0 version in official docs however it contains counting and listing tokens because works fine with 1.57.0 😄 https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/list-token

Reply:
+1

Reply:
`google-cloud-aiplatform = ">=1.57.0"` now in the project.toml